cypress/integration/stories/mla.spec.ts

```diff
@@ -0,0 +1,113 @@
+// Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {ClustersPage} from '../../pages/clusters.po';
+import {ProjectsPage} from '../../pages/projects.po';
+import {WizardPage} from '../../pages/wizard.po';
+import {login, logout} from '../../utils/auth';
+import {Condition} from '../../utils/condition';
+import {Preset} from '../../utils/preset';
+import {Datacenter, Provider} from '../../utils/provider';
+import {View} from '../../utils/view';
+import {WizardStep} from '../../utils/wizard';
+import * as _ from 'lodash';
+
+xdescribe('MLA Story', () => {
```

Comment:
I see that it's in review. Is this turned off for some reason? Besides that the test code looks fine, we can merge once the tests will pass.

Reply:
no that was not intended 😅 
However, it seems that I again need to take a look into the ci kubermatic configuration, as I'm currently getting `make: *** [Makefile:63: run-e2e-ci] Error 1` 🤔

Reply:
Sure, ping me for review/approval.